frontend/src/pages/team/Devices/dialogs/components/DeviceCredentialsDialog/OtcInstallSection.vue

```diff
@@ -88,6 +85,60 @@ export default {
         }
     },
     emits: ['select-os'],
+    computed: {
+        ...mapGetters('account', ['settings']),
+        commands () {
+            return {
+                script: {
+                    Windows: {
+                        title: 'Open Command Prompt as administrator and run:',
```

Comment:
```suggestion
                        title: 'Open an elevated Command Prompt and run:',
```

 "Elevated" is the standard and understood term for running an application with administrator privileges in Windows. It avoids the need to state "as administrator" (which usually means an account with admin rights) when it is paired with "Command Prompt."

An administrator account may not have unfettered access to everything. Whereas an elevated command prompt "will work"